pkg/app/app.go

```diff
@@ -1320,7 +1320,7 @@ func (a *App) apply(r *Run, c ApplyConfigProvider) (bool, bool, []error) {
 	st := r.state
 	helm := r.helm
 
-	helm.SetExtraArgs(argparser.GetArgs(c.Args(), r.state)...)
+	helm.SetExtraArgs(argparser.GetArgs(c.Args()+" "+c.DiffArgs(), r.state)...)
```

Comment:
@jkroepke please dont use `+`, try to use Join.

Reply:
done